src/query-builder/QueryBuilder.ts

```diff
@@ -624,12 +624,12 @@ export abstract class QueryBuilder<Entity> {
 
             if (replacementKeys.length) {
                 statement = statement.replace(new RegExp(
-                    `(?<=[ =\(]|^.{0})` +
+                    `([ =\(]|^.{0})` +
```

Comment:
Unrelated to this change but while we're here what is the effect of the `.{0}`?

Reply:
Not sure, seems to have originated [here](https://github.com/typeorm/typeorm/commit/fe94b8fa5b7935e6c82161486e9958ff53f82790).

Reply:
considering the code-change: since this is not reverting the code to the last known working state but "just" removing the lookbehind expression from the regex - can you comment on how this code still accomplishes what it should do?
And maybe we should add a code-comment here about the regex-compatibility issue so that the next guy is not introducing the same thing again.

Reply:
The [previous code](https://github.com/typeorm/typeorm/blob/2bf15ca913016ad07080c38c9fc3ee848b60ca4f/src/query-builder/QueryBuilder.ts#L574) did the same thing with the replacement. Lookahead/behind are just a simplification. I added a comment.